components/chart/description/index.tsx

```diff
@@ -55,52 +67,48 @@ const StyledDrawer = styled.div`
     padding-bottom: 10px;
   }
 `;
+const Drawer: React.FC<DrawerProps> = ({ open, onClose, module, children, updateURLWithNodeID }: DrawerProps) => {
+    const drawerRef = useRef(null);
+    const descriptionPanelRef = useRef(null);
 
-const Drawer: React.FC<PropsWithChildren<DrawerProps>> = ({
```

Comment:
Why you removed PropsWithChildren, without it building the project is not working :)

Reply:
Sorry By mistake